bin/interpreter.sh

```diff
@@ -42,6 +42,9 @@ while getopts "hp:d:l:v" o; do
             . "${bin}/common.sh"
             getZeppelinVersion
             ;;
+        u)
+            ZEPPELIN_SSH_COMMAND="ssh ${OPTARG}@localhost "
```

Comment:
How about make ZEPPELIN_SSH_COMMAND configurable? because some user may run ssh in non standard port, some user may event want to use different command, like sudo, to impersonate.

The idea is, we can make this part just get impersonate user
```bash
u)
    ZEPPELIN_IMPERSONATE_USER="${OPTARG}"
```

and then we can define ZEPPELIN_IMPERSONATE_CMD from `bin/common.sh` in configurable way

```bash
if [[ -z "$ZEPPELIN_IMPERSONATE_CMD" ]]; then
  ZEPPELIN_IMPERSONATE_CMD=`echo "ssh ${ZEPPELIN_IMPERSONATE_USER}@localhost"`
else
  ZEPPELIN_IMPERSONATE_CMD=$(eval "echo ${ZEPPELIN_IMPERSONATE_CMD}")
fi
```

So user can override ZEPPELIN_IMPERSONATE_CMD in `conf/zeppelin-env.sh` or inside of interpreter property, for example

```
ZEPPELIN_IMPERSONATE_CMD='sudo -u ${ZEPPELIN_IMPERSONATE_USER}'
```

what do you think?